tool/speed.cc

```diff
@@ -1790,8 +1866,9 @@ bool Speed(const std::vector<std::string> &args) {
      !SpeedRandom(selected) ||
      !SpeedECDH(selected) ||
      !SpeedECDSA(selected) ||
-     // OpenSSL 1.0 doesn't support Scrypt
 #if !defined(OPENSSL_1_0_BENCHMARK)
```

Comment:
Some of these curves are surely supported by 1.0.2?

Reply:
yes, the problem with 1.0.x is it doesn't support `BN_rand_range` and `EC_GROUP_get0_order` functions. That's why I left it out. I don't think the added complexity would be worth it (we don't care about performance of these "low level" functions with 1.0.x).

Reply:
You may to extend the comment on Scrypt, or in a separate one, to contain what you stated above.